### ISSUE TITLE
Bug 59405309: [React Native Gallery Preview]: Visually appearing heading 'React native gallery' is not defined as a heading programmatically.

### ISSUE THREAD

**Issue by vineethkuttan** [microsoft/react-native-gallery#699]
Build Details:
OS Version: 24H2 
OS Build: 26100.6584
App: React Native Gallery Preview
App Version: 1. 0.18.0
AT Used: Narrator

Repro-Steps:
Launch the React Native Gallery-Preview app.
Navigate to the 'Expand menu' button and invoke it.
Navigate to ‘Home' button present in left navigation pane and invoke it.
Now 'React native gallery' home page will be appeared
Now try to navigate to the 'React native gallery' heading using caps arrow and arrow key navigation
Now listen to the Narrator announcement
Actual Result:
Visually appearing heading 'React native gallery' is not defined as a heading programmatically. While navigating to it, Screen reader is announcing as normal text. i.e., 'React native gallery'

Note: Same issue is observed all the headings available in entire application

Expected Result:
Visually appearing heading 'React native gallery' should be defined as a heading. While navigating to it, Screen reader should convey as 'React native gallery, heading level 1'.

User Impact:
Users who rely on Screen reader will face difficulty if visually looking heading is not defined as heading.

MAS Reference Link: [MAS 1.3.1 – Info and Relationships](https://aka.ms/MAS1.3.1)

Trap ID: [1.6 - Poor Grouping](https://aka.ms/PoorGrouping)

“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”

**Comment by vineethkuttan** [comment]
We have to accessibiltyLevel For all headings in all pages

**Comment by protikbiswas100** [comment]
https://github.com/microsoft/react-native-windows/pull/15193
